@akoeplinger this looks related to enabling Swift linking on iOS that was done in https://github.com/dotnet/runtime/pull/104383.

```
/tmp/helix/working/BA9A0A0E/p/build/apple/AppleBuild.targets(301,5): error MSB4018: ld: warning: Could not find or use auto-linked library 'swiftXPC' [/private/tmp/helix/working/BA9A0A0E/w/A6B50900/e/publish/ProxyProjectForAOTOnHelix.proj]
/tmp/helix/working/BA9A0A0E/p/build/apple/AppleBuild.targets(301,5): error MSB4018: ld: warning: Could not find or use auto-linked library 'swiftObservation' [/private/tmp/helix/working/BA9A0A0E/w/A6B50900/e/publish/ProxyProjectForAOTOnHelix.proj]
/tmp/helix/working/BA9A0A0E/p/build/apple/AppleBuild.targets(301,5): error MSB4018: Undefined symbols for architecture arm64: [/private/tmp/helix/working/BA9A0A0E/w/A6B50900/e/publish/ProxyProjectForAOTOnHelix.proj]
/tmp/helix/working/BA9A0A0E/p/build/apple/AppleBuild.targets(301,5): error MSB4018:   "__swift_FORCE_LOAD_$_swiftXPC", referenced from: [/private/tmp/helix/working/BA9A0A0E/w/A6B50900/e/publish/ProxyProjectForAOTOnHelix.proj]
/tmp/helix/working/BA9A0A0E/p/build/apple/AppleBuild.targets(301,5): error MSB4018:       __swift_FORCE_LOAD_$_swiftXPC_$_pal_swiftbindings in libSystem.Security.Cryptography.Native.Apple.a(pal_swiftbindings.o) [/private/tmp/helix/working/BA9A0A0E/w/A6B50900/e/publish/ProxyProjectForAOTOnHelix.proj]
```

Any chance something jumps out at you as the cause of the issue?

--- a/eng/pipelines/common/xplat-setup.yml
+++ b/eng/pipelines/common/xplat-setup.yml
@@ -170,7 +170,7 @@ jobs:
 
         # OSX Public Build Pool (we don't have on-prem OSX BuildPool).
         ${{ if and(in(parameters.osGroup, 'osx', 'maccatalyst', 'ios', 'iossimulator', 'tvos', 'tvossimulator'), eq(variables['System.TeamProject'], 'public')) }}:
-          vmImage: 'macos-latest'
+          vmImage: 'macos-13'
 
         # Official build OSX pool
         ${{ if and(in(parameters.osGroup, 'osx', 'maccatalyst', 'ios', 'iossimulator', 'tvos', 'tvossimulator'), ne(variables['System.TeamProject'], 'public')) }}:

--- a/eng/pipelines/coreclr/perf-non-wasm-jobs.yml
+++ b/eng/pipelines/coreclr/perf-non-wasm-jobs.yml
@@ -374,7 +374,7 @@ jobs:
         nameSuffix: PerfBDNApp
         isOfficialBuild: false
         pool:
-          vmImage: 'macos-latest'
+          vmImage: 'macos-13'
         postBuildSteps:
           - template: /eng/pipelines/coreclr/templates/build-perf-bdn-app.yml
             parameters:

